app/representers/account_representer.rb

```diff
@@ -67,5 +70,9 @@ def credit_card_stored
     credit_card_stored?
   end
 
+  def admin_name
+    account.admins.first.try(:display_name)
```

Comment:
You may want to use one of these methods instead of doing `.admins.first` here 😄 
https://github.com/3scale/porta/blob/23c541044bf72ea2e9632129877ef0d393e9e32c/app/models/account.rb#L192-L198